source/Buoy-Localization-GS64-Extensions/CharacterCollection.extension.st

```diff
@@ -0,0 +1,55 @@
+Extension { #name : 'CharacterCollection' }
+
+{ #category : '*Buoy-Localization-GS64-Extensions' }
+CharacterCollection >> asLanguageRange [
+
+  ^ LanguageRange fromString: self
+]
+
+{ #category : '*Buoy-Localization-GS64-Extensions' }
+CharacterCollection >> asLanguageTag [
+
+	^ LanguageTag fromString: self
+]
+
+{ #category : '*Buoy-Localization-GS64-Extensions' }
+CharacterCollection >> escaped [
+
```

Comment:
Create a separate class EscapingAlgorithm and put the code there, to prevent reimplementing this twice